stable/mongodb/values-production.yaml

```
@@ -73,6 +73,7 @@ clusterDomain: cluster.local
 service:
   annotations: {}
   type: ClusterIP
+  #clusterIP: None
```

Sorry for being picky, can you add a space between the `#` and `clusterIP`? to be consistent with other values:
```suggestion
  # clusterIP: None
```

---

done. Thanks,